tests/Php54/Php54Test.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Polyfill\Tests\Php54;
+
+use Symfony\Polyfill\Php54\Php54 as p;
+
+class Php54Test extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @runInSeparateProcess
+     */
+    public function testHttpResponseCodeDefaultValueIs200() {
+        $code = p::http_response_code(null);
```

Comment:
> should be against the native function 

That won't work.  This function behaves differently under CLI.  We'd want to compare the CGI/FPM versions, which we obviously cannot do using PHPUnit.